nodejs/advanced/Demo.Data.Stream.Processing/library.json

```diff
@@ -1,6 +1,6 @@
 {
     "libraryItemId": "a1c46790-6e68-4d7a-8ce2-18ae414b6bf7",
-    "name": "Streaming Demo - UI",
+    "name": "Streaming demo UI",
```

Comment:
> "name": "Streaming demo UI",

Didn't we decide to call it `Sentiment Demo UI`? Or are we referring to something else here? If something else, just want to check it should be in this PR?

Reply:
This is the name that will show in the library